lib/execution_engine2/SDKMethodRunner.py

```diff
@@ -835,9 +835,14 @@ def check_jobs(self, job_ids, check_permission=True, projection=None):
 
             job_states[str(job.id)] = mongo_rec
 
+        job_states = dict(sorted(job_states.items()))
```

Comment:
Does this return in the order that the jobs  were requested? For example

1000,500,300 would return the records as 1000,500,300 and not as 300,500,1000

Reply:
💯 added sort_by to `get_jobs`